x-pack/plugins/siem/public/lib/telemetry/index.ts

```diff
@@ -25,9 +27,9 @@ export const track: TrackFn = (type, event, count) => {
 
 export const initTelemetry = (usageCollection: SetupPlugins['usageCollection'], appId: string) => {
   try {
-    _track = usageCollection.reportUiStats.bind(null, appId);
+    _track = usageCollection?.reportUiStats.bind(null, appId) ?? noop;
   } catch (error) {
-    // ignore failed setup here, as we'll just have an inert tracker
+    _track = noop;
```

Comment:
Can this part of the exception be hit anymore? It doesn't seem like it with the way you changed it above. Just a question I have.

Reply:
I had a similar thought! Probably not, unless `reportUiStats` wasn't a function? I debated between the safety of the try/catch vs the cleanliness of not having it.